runtime/server/mcp.go

```diff
@@ -226,6 +235,8 @@ func (s *Server) mcpQueryMetricsView() (mcp.Tool, server.ToolHandlerFunc) {
 Perform an arbitrary aggregation on a metrics view.
 Tip: Use the 'sort' and 'limit' parameters for best results and to avoid large, unbounded result sets.
 Important note: The 'time_range' parameter is inclusive of the start time and exclusive of the end time.
+Note: 'time_dimension' is an optional parameter under "time_range" that can be used to specify the time dimension to use for the time range. 
+If not provided, the default time column of the metrics view will be used.
```

Comment:
Keep it as one line so AI knows the second sentence is related to the first
```suggestion
Note: 'time_dimension' is an optional parameter under "time_range" that can be used to specify the time dimension to use for the time range.  If not provided, the default time column of the metrics view will be used.
```